frontend/src/components/ShootWorkers/ContainerRuntime.vue

```diff
@@ -0,0 +1,192 @@
+
+<!--
+SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <div class="d-flex flex-row">
+    <v-select
+      color="primary"
+      item-color="primary"
+      :items="containerRuntimeItems"
+      :error-messages="getErrorMessages('containerRuntime')"
+      @input="onInputContainerRuntime"
+      @blur="$v.containerRuntime.$touch()"
+      v-model="containerRuntime"
+      label="Container Runtime"
+      :disabled="immutableCri"
+      :hint="criHint"
+      persistent-hint
+    >
+      <template v-slot:item="{ item }">
+        <span v-if="!immutableCri && item===defaultCir">default ({{item}})</span>
+        <span v-else>{{item}}</span>
+      </template>
+      <template v-slot:selection="{ item }">
+        <span v-if="!immutableCri && item===defaultCir">default ({{item}})</span>
+        <span v-else>{{item}}</span>
+      </template>
```

Comment:
```suggestion
```suggestion
      <template v-slot:item="{ item }">
        <span>{{criItemText(item)}}</span>
      </template>
      <template v-slot:selection="{ item }">
        <span>{{criItemText(item)}}</span>
      </template>
```
```